<launch>
    <arg name="model_path" default="$(find-pkg-share semantic_segmentation)/data/model_float32.pb"/>

    <!-- ========================================= -->
    <!-- ============ PARTICLE FILTER ============ -->
    <!-- ========================================= -->
    <node name="semseg_node" pkg="semantic_segmentation" exec="semseg_node.py" output="screen" args="--ros-args --log-level info">
        <param name="model_path" value="$(var model_path)"/>
        <remap from="/in/image_raw" to="/sensing/camera/undistorted/image_raw"/>
        <remap from="/out/image_raw" to="/segmseg/image_raw"/>
    </node>
</launch>
